tensorflow/java/BUILD

```diff
@@ -106,6 +106,7 @@ cc_binary(
     # symbols from the library. This reduces the size of the library
     # considerably (~50% as of January 2017).
     linkopts = select({
+        "//tensorflow:debug" : [], # Disable all custom linker options in debug mode
```

Comment:
```
FAIL: buildifier found errors and/or warnings in above BUILD files.
buildifier suggested the following changes:
110c110
<         "//tensorflow:debug" : [], # Disable all custom linker options in debug mode
---
>         "//tensorflow:debug": [],  # Disable all custom linker options in debug mode
Please fix manually or run buildifier <file> to auto-fix.
```
Two spacing errors here. No space between `...debug"` and `:`. Then exactly two spaces between `[],` and `#`